01_separate_projects/flow_codegen/tests/test_metamodel.py

```diff
@@ -1,18 +1,35 @@
 from __future__ import unicode_literals
 import os
+from textx.cli import textx
+from click.testing import CliRunner
+
+
+this_folder = os.path.abspath(os.path.dirname(__file__))
 
 
 def test_flow_dsl():
-    import flow_dsl
-    mmF = flow_dsl.get_metamodel_flow()
-    current_dir = os.path.dirname(__file__)
-    model = mmF.model_from_file(os.path.join(current_dir,
-                                             'models',
-                                             'data_flow.flow'))
-    assert(model is not None)
+    fname = os.path.join(this_folder,
+                         'models', 'data_flow.pu')
+    try:
+        os.remove(fname)
+    except OSError:
+        pass
+
+    model_file = os.path.join(this_folder,
+                              'models', 'data_flow.eflow1')
```

Comment:
Why do we need `1` suffix for all model files? It looks artificial.

Reply:
Oh, I get it now. You are enabling all three approaches to be installed/tested together. That makes sense.

Reply:
(no action)